lms/djangoapps/instructor/views/api.py

```diff
@@ -3058,11 +3056,7 @@ def generate_certificate_exceptions(request, course_id, generate_for=None):
             status=400
         )
     else:
-        students = User.objects.filter(
-            id__in=users,
-            certificatewhitelist__course_id=course_key,
-            certificatewhitelist__whitelist=True
-        )
+        students = 'whitelisted_not_generated'
```

Comment:
@adampalay this code depends on the students coming from the browser first one is for all whitelisted students